server/ansible/Inventory/pbench-server.hosts.example

```diff
@@ -1,9 +1,23 @@
+# Change the [servers] section to the set of hosts where you want to
+# install pbench-server. The various hosts can be running any
+# combinations of RHEL7, RHEL8, or (supported) Fedora versions
+# (currently 28 and 29).  They will also have different configuration
+# files which can be specifified using the value of the cenv variable
+# (which is used to set the value of the
+# pbench_configuration_environment variable which is use to retrieve
```

Comment:
Consider s/variable which is use to retrieve/variable, which is used to retrieve/

Reply:
Done. The doc as a whole has been modified, reorganized and clarified.